### ISSUE TITLE
Query Scratch Pad is visible by default even if closed once before

### ISSUE THREAD

**Issue by wuestkamp** [pgadmin-org/pgadmin4#7919]
**Describe the bug**

Query Scratch Pad is visible by default even if closed once before.
This is the case in `docker.io/dpage/pgadmin4:8.11`

This does not happen in `docker.io/dpage/pgadmin4:8.2`.
There once the Query Scratch Pad was closed once, it won't be shown automatically if a new Query Window is opened.

It would be great if this could be reverted or if a setting exists that allows to disable the Query Scratch Pad?

**Comment by anilsahoo20** [comment]
Tested and verified on snapshot build: https://www.postgresql.org/ftp/pgadmin/pgadmin4/snapshots/2024-10-03/
Package: arm64, pip wheel
Environment: macOs Ventura 13.5.1

**Comment by EcBen** [comment]
@anilsahoo20 I tested the latest build from yesterday to fix this issue in 9.5:
- https://github.com/pgadmin-org/pgadmin4/issues/8918

Using this build:
- https://github.com/pgadmin-org/pgadmin4/issues/8901#issuecomment-3112609209

That resolved those issues, but this Scratch Pad persisting issue has returned in the nightly build. If I click the `X` to close the `Scratch Pad` tab/panel, close the Query Tool window. Then open a new Query Tool window, the Scratch Pad tab/panel is back. The Scratch Pad tab/panel should not return after I've closed it.
